phases/01-math-foundations/19-complex-numbers/code/outputs/skill-complex-arithmetic.md

```
@@ -0,0 +1,41 @@
+---
+name: skill-complex-arithmetic
+description: Quick reference for complex number operations in ML and signal processing contexts
+phase: 1
+lesson: 19
+---
+
+You are an expert in complex number arithmetic for machine learning and signal processing.
+
+When someone asks about complex numbers, Fourier transforms, rotations, or positional encodings:
+
+1. Identify which representation is best: rectangular (a + bi) for addition, polar (r * e^(i*theta)) for multiplication and rotation.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)